nUpdate.WPFUserInterface/ServiceInterfaces/IDialogWindowService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using nUpdate.ViewModel.Interfaces;
+
+namespace nUpdate.ServiceInterfaces
```

Comment:
Every namespace in nUpdate.WPFUserInterface should begin with the project name. For example, this should be "nUpdate.WPFUserInterface.ServiceInterfaces".